File_Engine/Modify/Rename.cs

```diff
@@ -47,15 +47,19 @@ public static IFSContainer IRename(this oM.Adapters.File.IFSContainer fileOrDir,
             return Rename(fileOrDir as dynamic, name);
         }
 
+        /***************************************************/
+        /**** Private interface methods                 ****/
         /***************************************************/
 
-        public static IFSContainer Rename(this FSFile file, string name)
+        private static IFSContainer Rename(this FSFile file, string name)
         {
             file.Name = name;
             return file;
         }
 
-        public static IFSContainer Rename(this FSDirectory directory, string name)
+        /***************************************************/
+
+        private static IFSContainer Rename(this FSDirectory directory, string name)
```

Comment:
These require versioning @Tom-Kingstone

Reply:
They class as 'removed' methods.

Optionally, you could use `PreviousVersion` on the `IRename` method instead to say they should go to that method.

Reply:
We also require a `Rename(this object obj)` fallback method to avoid file crashes where something implements `IFSContainer` later but doesn't add a method here.